packages/clerk-js/src/utils/web3.ts

```diff
@@ -1,10 +1,15 @@
+import type { Web3Provider } from '@clerk/types';
+
 import { toHex } from './hex';
 
-export async function getMetamaskIdentifier(): Promise<string> {
+type GetWeb3IdentifierParams = {
+  provider: Web3Provider;
+};
+
+export async function getWeb3Identifier(_: GetWeb3IdentifierParams): Promise<string> {
   // @ts-ignore
   if (!global.ethereum) {
-    // Do nothing when ethereum doesn't exist. We might revise this in the future
-    // to offer an Install Metamask prompt to our users.
```

Comment:
Is this recommendation not viable ? (just curious)

Reply:
Not sure if it's possible or what's the recommended way (hopefully we will find out soon), but removed the comment until we have a concrete plan in place